### DISCUSSION TITLE
Hardening against fingerprinting

### DISCUSSION THREAD

**Discussion by gurre** [authelia/authelia#8564] [Documentation Feedback]
I've noticed that the frontend code is prepared with a `showBrand`-prop. Can this be configured so the brand can be hidden?

Further I'm interested if there is in any other way authelia may be fingerprinted for version or reveal itself.
 - Is the version exposed in the API?
 - Does the API differ between versions in way that would uniquely identity it?

**Comment by gurre** [comment]
A possible way of leaking the service name is through the default cookie name.

```
session:
  # The name of the session cookie. (default: authelia_session).
  name: authelia_session
```

**Comment by james-d-elliott** [comment]
1. Version is not exposed anywhere. To my knowledge. Definitely not via the API.
2. The API does differ between versions, however the public portion of that has not changed for about 4 months and is not expected to change again.
3. Yes the session is a possible fingerprint. It's easy to change.

We should maybe make this into a discussion around a document for the site.
